webview/src/setup/components/remote/MultiProjectRemotes.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { RemoteList } from 'dvc/src/setup/webview/contract'
+import styles from './styles.module.scss'
+
+export const MultiProjectRemotes: React.FC<{
+  remoteList: NonNullable<RemoteList>
+}> = ({ remoteList }) => (
+  <table data-testid="remote-details" className={styles.remoteDetails}>
+    <tbody>
```

Comment:
Minor, but we technically don't need this `tbody` in the `table` since a `tbody` is only needed if `table` has `thead`, `tfoot` or multiple `tbody`s. We also have an unneeded `tbody` in dvc env table... I'll change that :)